aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -92,9 +92,14 @@ func resourceAviatrixSpokeGateway() *schema.Resource {
 					"Otherwise, allocate a new Elastic IP and use it for this gateway.",
 			},
 			"eip": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Computed:    true,
+				Type:     schema.TypeString,
+				Optional: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					if old != "" {
+						return new != old
+					}
+					return false
+				},
```

Comment:
Could you explain the reasoning for this? We will never show a diff after the eip is set?

Also nitpick: could be one line `return old != "" && new != old`

Reply:
For this feature, we don't allocate new eip for the tgw, so we have to set eip. But I found that no matter what you set, the backend will assign a different one (not a new eip), which will trigger a diff.

Reply:
Have you asked Steve about that behavior? I think we can find a better solution than just never reading a diff for this attribute

Reply:
Now I stop reading any diff for `allocate_new_eip` if private oob is enabled.